nopol/src/main/java/fr/inria/lille/spirals/evo/processors/RemoveEvosuiteEffectsProcessor.java

```diff
@@ -0,0 +1,65 @@
+package fr.inria.lille.spirals.evo.processors;
+
+import java.lang.annotation.Annotation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import spoon.processing.AbstractProcessor;
+import spoon.reflect.declaration.CtAnnotation;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.declaration.CtElement;
+import spoon.reflect.declaration.CtMethod;
+
+public class RemoveEvosuiteEffectsProcessor extends AbstractProcessor<CtClass>{
+
+	private static final ArrayList<String> UnwantedAnnotations = new ArrayList<String>();
```

Comment:
The convention in java is to use camel case, here it's a constant you can also put it in upper-case